docs/docsite/rst/vmware_scenarios/vmware_inventory_vm_attributes.rst

```diff
@@ -312,7 +312,7 @@ guestId (str)
 alternateGuestName (str)
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-    Used as display name for the operating system if guestId isotherorother-64. See :ref:`guest_full_name`.
+    Used as display name for the operating system if guestId isotherorother-64. See :ref:`ansible_collections.community.vmware.docsite.guest_full_name`.
```

Comment:
Not really related to this PR, but "if guestId isotherorother-64" sounds pretty strange to me.

Reply:
Not only to you :-)

The scenario guides have been moved from Ansible 2.9 to this collection, but there hasn't been much work invested since. I think I'll keep this one as it is. Hopefully, someone will find the time to overhaul them and fix this.

I've added a changelog fragment and will merge as-is to fix the broken links. It's at least some improvement.